libs/WordPressProcessors/build.gradle

```diff
@@ -0,0 +1,24 @@
+apply plugin: 'kotlin'
+apply plugin: 'kotlin-kapt'
+
+ext.kotlinVersion = '1.3.61'
+
+kapt {
+    generateStubs = true
+}
+
+dependencies {
+    implementation project(':libs:WordPressAnnotations')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    implementation "com.google.auto.service:auto-service:1.0-rc6"
```

Comment:
We might want to consider defining all the dependencies' versions in the main build.gradle - [here](https://github.com/wordpress-mobile/WordPress-Android/blob/4c855834299c176a2711e3af22efc1183c4252b9/build.gradle#L116). Wdyt?